packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/GitHub/index.tsx

```diff
@@ -53,20 +54,24 @@ export const GitHub = () => {
         isFetching,
         isExported,
         pr,
+        isEnablingSync,
```

Comment:
Maybe this is a bit ambiguous for other readers, we could maybe name this to `isLinkingToGitSandbox`?

Reply:
true, I made this when I was calling git sync